config/Dockerfiles/tests.d/combined/01_beaker_openstack

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 
 # Provision both beaker and openstack with one PinFile
-# distros.exclude: fedora28 fedora29 fedora30 centos7
+# distros.exclude: fedora29 fedora30 centos7
```

Comment:
This test has been disabled for a while. I'm opening #1183 to resolve that

Reply:
Perfect !